### ISSUE TITLE
Any plans for a model registry? just like mlflow

### ISSUE THREAD

**Issue by GeeCastro** [aimhubio/aim#3234]
## 🚀 Feature

A model registry that links to the run it was produced with. And the capability of promoting a run to a model version.

### Motivation

It's quite crucial to keep a model registry. A record of the successful models that were candidates for production. Ideally with a history of different versions.

**Comment by sbatchelder** [comment]
It's not the same as a proper model registry, but for each run I've been saving the model checkpoints and onnx model exports as a Run Artifact. This makes the models for a run accessible an a way that's sufficient for my needs, but ymmv.

**Comment by michalsustr** [comment]
Agreed, you may want to have a look at https://minfx.ai - we are building such features.
